proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -171,3 +180,19 @@ message ProposedChain {
   string chainID = 1;
   uint64 proposalID = 2;
 }
+
+message QueryAllPairsValConAddrByConsumerChainIDRequest {
+  // The id of the consumer chain
+  string chain_id = 1;
+}
+
+message QueryAllPairsValConAddrByConsumerChainIDResponse {
+  repeated PairValConAddrProviderAndConsumer pair_val_con_addr = 1;
+}
+
+message PairValConAddrProviderAndConsumer {
```

Comment:
Would you mind also adding the public key to this struct?

All of the addresses are returned with `cosmos...` bech prefix, so having the public key available makes it easier to figure out the address on the consumer chain.

Reply:
I added it, plz check